python/tvm/topi/cuda/conv2d_transpose_nchw.py

```diff
@@ -194,6 +197,8 @@ def _callback(op):
 
             if cfg.is_fallback:
                 N, F, Y, X = get_const_tuple(conv.shape)
+                if not isinstance(N, int):
+                    N = 1
```

Comment:
Can we add a test that hits this change?

Reply:
Yeah we do have a test for this. Now I enabled all targets.